api/app/src/shared/notifications.ts

```diff
@@ -48,14 +50,6 @@ export const sendNotification = async (notif: SlackMessage | string): Promise<vo
 export const sendAlert = async (notif: SlackMessage | string): Promise<void> =>
   sendToSlack(notif, slackAlertUrl);
 
-export type CaptureContext = {
-  user: { id?: string; email?: string };
-  extra: Record<string, unknown>;
-  tags: {
-    [key: string]: string;
-  };
-};
```

Comment:
No need to define the type, we can access it from Sentry through `@sentry/types`

Using the right type also allows us to set the `level` of the report (`'fatal' | 'error' | 'warning' | 'log' | 'info' | 'debug'`).